app/views/items/_category_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<p>
+  <%= form_for Category.new, html: { class: 'form-inline' } do |f| %>
```

Comment:
The better way to do it is to create some sort of presentation object with everything you need.  Just some PORO like ItemIndexPresenter that manages the three different object to expose.